### PR TITLE
More Responsive Fix and Style Updates

### PR DESCRIPTION
**Pull request by francepastrana**

### CODE REVIEW

File: src/styles/OrderSummary.css [hunk 1]
```diff
@@ -138,23 +138,29 @@ span.price--extra-addon {
     bottom: 0;
     position: relative;
     right: 0;
-    width: 407px;
+    width: 87%;
```
Comment (JPTrey91) [inline src/styles/OrderSummary.css:141]: If we're going to use percentages, we should also have `min` and `max-width` set.

**Review by JPTrey91 — CHANGES_REQUESTED** [review]
OrderSummary is misaligned when simulating an iPhone 14 Pro.

Screen Shot 2022-10-31 at 2 14 27 PM

**Review by JPTrey91 — APPROVED** [review]

**Review by JPTrey91 — CHANGES_REQUESTED** [review]
Still seeing visual issues: 
Screen Shot 2022-11-02 at 11 38 47 AM

**Comment by francepastrana** [comment]
> Still seeing visual issues: Screen Shot 2022-11-02 at 11 38 47 AM

image

Floating order summary and header display condition fixed

**Comment by JPTrey91** [comment]
Still seeing too many visual inconsistencies to approve this.  Please revise: 

Screen Shot 2022-11-03 at 10 42 53 AM

Screen Shot 2022-11-03 at 10 44 42 AM

Screen Shot 2022-11-03 at 10 44 47 AM

Screen Shot 2022-11-03 at 10 45 02 AM

**Comment by francepastrana** [comment]
@JPTrey91 Weird. Dish card modals on mobile is already fixed. Something is not right here. Anyway I re-fixed it again and added style elements to match figma.

**Review by JPTrey91 — APPROVED** [review]

**Pull request merged**